scripts/diagnostics.txt

```diff
@@ -543,6 +545,7 @@ note InfoTask "$info encountered{}"
 note NoteComparisonReduces "comparison reduces to ({} {} {})"
 note NoteCommonAncestor "common ancestor here violates that constraint"
 warning explicit-static StaticInitializerMustBeExplicit "initializing a static variable in a procedural context requires an explicit 'static' keyword"
+note VirtualInterfaceDecl "virtual interface declared here"
```

Comment:
In general we don't need to keep adding new notes that are very slight variations of existing ones; the more general "declared here" note is fine for these purposes.

Reply:
removed